game/prefetch.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
@mvl22, @Siequnu -- could something like this work? Set aside an invisible div for loading abst in the background. It'll inject a canvas element with the full window size once it loads. When we click the "Simulation" button, we can flip-flop visibility of some divs.